lib/tracker_api/endpoints/attachment.rb

```diff
@@ -0,0 +1,37 @@
+module TrackerApi
+  module Endpoints
+    class Attachment
+      attr_accessor :client
+
+      def initialize(client)
+        @client = client
+      end
+
+      def create(comment, file)
+        data = client.post("/projects/#{comment.project_id}/uploads", body: FileUtility.get_file_upload(file)).body
+        Resources::FileAttachment.new({ comment: comment }.merge(data))
+      end
+
+      # This orphans the file, deletes source, but appears in the comments
```

Comment:
Can we remove the commented out code?

You can put TODOs or notes in comments at the top of the class if you want.